### PR TITLE
f-tabs@v0.2.1 - Unit tests

### PR DESCRIPTION
**Pull request by matthewhardern**
Adds unit tests for the tabs component plus adds some spacing for readability to keep more inline with other components.

### CODE REVIEW

File: packages/components/molecules/f-tabs/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v0.2.1
+------------------------------
+*February 12, 2021*
```
Comment (ashleynolan) [inline packages/components/molecules/f-tabs/CHANGELOG.md]: ⌛ Not sure if you're just feeling pessimistic about when you'll get some reviews 😆

File: packages/components/molecules/f-tabs/src/components/tests/Tabs.test.js [hunk 2]
```diff
@@ -1,10 +1,122 @@
 import { shallowMount } from '@vue/test-utils';
-import VueTabs from '../Tabs.vue';
+import Tabs from '../Tabs.vue';
+
+const registeredTabs = [
```
Comment (ashleynolan) [inline packages/components/molecules/f-tabs/src/components/tests/Tabs.test.js]: Not sure that `registeredTabs` is the best name for this – feels like more like mock data to me, so maybe `tabMocks` or something like that (there's probably a better name out there, but my mind is blank atm!)

File: packages/components/molecules/f-tabs/CHANGELOG.md (hunk 1 above)
Comment (matthewhardern) [reply packages/components/molecules/f-tabs/CHANGELOG.md]: lol, thats the day i am having. I get it changed to the actual date

File: packages/components/molecules/f-tabs/src/components/tests/Tabs.test.js (hunk 2 above)
Comment (matthewhardern) [reply packages/components/molecules/f-tabs/src/components/tests/Tabs.test.js]: yes agreed will make the change

Comment (matthewhardern) [reply packages/components/molecules/f-tabs/src/components/tests/Tabs.test.js]: changed to `registeredTabsMock`

**Review by ashleynolan — APPROVED** [review]

File: packages/components/molecules/f-tabs/src/components/tests/Tab.test.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            });
+
+            // Assert
+            expect(leftWrapper.vm.transitionName).toEqual('fade-in-right');
+        });
+
+        it('should apply the fade-in-left class when animationDirection is RIGHT', () => {
+            // Arrange
+            const rightWrapper = shallowMount(Tab, {
+                propsData: {
```
Comment (FayeCarter) [inline packages/components/molecules/f-tabs/src/components/tests/Tab.test.js:113]: Could this be `propsData: REGISTER_DATA`

**Review by dandel10n — APPROVED** [review]

**Pull request merged**